cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResources.java

```diff
@@ -841,50 +842,48 @@ public CloudEncryptionKeys encryptionKeys(ExtendedCloudCredential cloudCredentia
         CloudEncryptionKeys cloudEncryptionKeys = new CloudEncryptionKeys(new HashSet<>());
         AwsCredentialView awsCredentialView = new AwsCredentialView(cloudCredential);
         AmazonKmsClient client = awsClient.createAWSKMS(awsCredentialView, region.value());
+        Set<KeyListEntry> kmsKeys = getListKeyResult(client);
+        Set<AliasListEntry> listAliasesResult = getListAliasResult(client);
```

Comment:
I believe these two lines 
```
Set<KeyListEntry> kmsKeys = getListKeyResult(client);
Set<AliasListEntry> listAliasesResult = getListAliasResult(client);
```
should be within the first try block for backward compatibility. Otherwise you may lose a valuable error message.